src/quick-tour/path-to-purchase.md

```diff
@@ -2,21 +2,24 @@
 title: Path to Purchase
 ---
 
-The path customers take that leads to a sale is sometimes called the “path to purchase.” In this quick tour, we’ll take a look at pages of strategic value that customers usually visit while shopping in your store. Then, we’ll consider different store features that can be leveraged at each stage of the customer journey.
+The path customers take that leads to a sale is sometimes called the _path to purchase_. In this quick tour, we take a look at pages of strategic value that customers usually visit while shopping in your store. We also consider different store features that can be leveraged at each stage of the customer journey.
 
-[<!--{% if "Default.CE Only" contains site.edition %}-->![]({{ site.baseurl }}{% link images/images/storefront-homepage.png %}){: .zoom}<!--{% endif %}--><!--{% if "Default.EE-B2B" contains site.edition %}-->![]({{ site.baseurl }}{% link images/images-ee/storefront-homepage-ee.png %}){: .zoom}<!--{% endif %}-->]({{ site.baseurl }}{% link quick-tour/home-page.md %})
+<!--{% if "Default.CE Only" contains site.edition %}-->![storefront home page]({{ site.baseurl }}{% link images/images/storefront-homepage.png %}){: .zoom}
+<!--{% endif %}-->
+<!--{% if "Default.EE-B2B" contains site.edition %}-->![storefront home page]({{ site.baseurl }}{% link images/images-ee/storefront-homepage-ee.png %}){: .zoom}
+<!--{% endif %}-->
 
-[Home Page]({{ site.baseurl }}{% link quick-tour/home-page.md %})
-:  Your home page is like the front window display of your store. As the primary landing page, its design entices visitors to come inside for a closer look.
+Home Page
+:  Your [home page]({{ site.baseurl }}{% link quick-tour/home-page.md %}) is like the front window display of your store. As the primary landing page, its design entices visitors to come inside for a closer look.
 
-[Catalog Page]({{ site.baseurl }}{% link quick-tour/catalog-page.md %})
-:  This page shows products from your catalog in either a list or grid format. The selection can be based on a category chosen from the main menu, a choice made in the layered navigation on the left, or the results of a search. Any item can be examined in more detail, or placed directly into the shopping cart.
+Catalog Page
+:  The [Catalog page]({{ site.baseurl }}{% link quick-tour/catalog-page.md %}) shows products from your catalog in either a list or grid format. Customers can make selections  based on a category they choose from the main menu, by using the layered navigation on the left, or from the results of a search. They can examine the item in more detail, or placed it directly into the shopping cart.
```

Comment:
1. remove , after "left"
2. "more detail or place it directly".   remove , and "d"